src/hotspot/os_cpu/bsd_aarch64/vm_version_bsd_aarch64.cpp

```diff
@@ -199,7 +199,12 @@ unsigned long VM_Version::os_get_processor_features() {
   size_t i;
   const struct cpu_parts *cpu_partsp = NULL;
 
+#ifdef __OpenBSD__
+  // READ_SPECIALREG is not available from userland on OpenBSD yet
```

Comment:
Should we just do

```
#ifdef __OpenBSD__
#define READ_SPECIALREG(x) 0
#endif
```

instead of wrapping all of the calls to it separately?

Reply:
I decided to hardcode the values for OpenBSD in a single block after discussing this with some kernel devs at OpenBSD.